zipline/finance/blotter/__init__.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .simulatedblotter import SimulatedBlotter
+from .core import (
+    blotters,
+    register,
+    unregister,
+    load,
+)
+
+__all__ = ['SimulatedBlotter', 'unregister', 'blotters', 'load']
+
+
+@register('default')
+def default_blotter():
```

Comment:
Is there a reason to wrap this in a function that returns the class instead of just registering the class directly?

Reply:
I wanted to wrap this so that like with metrics, we would store each blotter class as a function (and load it by calling that function), which would allow people to use decorators in general. It was basically just for consistency purposes, because otherwise I don't exactly know how we could use decorators.

Reply:
You can decorate a class directly, just like you would a function. For example:

```python
@register('my-blotter')
class MyBlotter(Blotter):
    ...
```